helm-chart/dash0-operator/values.yaml

```diff
@@ -290,6 +298,16 @@ operator:
     # override the default image pull policy
     pullPolicy:
 
+  # the container image to use for the target-allocator component (there should usually be no reason to override this)
+  targetAllocatorImage:
+    repository: "ghcr.io/open-telemetry/opentelemetry-operator/target-allocator"
+    # overrides the image tag, which defaults to the chart appVersion.
+    tag: "0.140.0"
```

Comment:
_Non-blocking:_ As a follow-up we probably should make sure we create PRs to update the version regularly. Either via Dependabot or by integrating it into  `.github/workflows/update-collector-components.yaml` or something similar.

Reply:
+1 for separate PR, definitely.